RefrigeratorExe/RefrigeratorExe/Refrigerator.cs

```diff
@@ -0,0 +1,185 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Globalization;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using System.Xml.Linq;
+
+namespace RefrigeratorExe
+{
+    internal class Refrigerator :IComparable<Refrigerator>
+    {
+        private string _model;
+
+
+        public Guid id { get; }
+        public string Model
+        {
+            get
+            {
+                return _model;
+            }
+            private set
+            {
+                if (!value.Equals(""))
+                    _model = value;
+                else
+                    throw new ArithmeticException("Illegal model ,model must contain a String ");
+            }
+        }
+        public string Color { get; set; }
+        public int NumberOfShelfs { get; set; }
+        public List<Shelf> Shelves;
+
+
+        public Refrigerator(string model, string color, int shelf)
+        {
+            id = Guid.NewGuid();
+            try
+            {
+                Model = model;
+
+            }
+            catch (Exception e)
+            {
+
+                Console.WriteLine(e.Message); 
+            }
+            Color = color;
+            NumberOfShelfs = shelf;
+            Shelves = new List<Shelf>();
+            this.AddShelfs();
+        }
+        public void AddShelfs()
+        {
+            while (this.Shelves.Count() < this.NumberOfShelfs)
+            {
+                this.Shelves.Add(new Shelf(this.Shelves.Count()));
+            }
+        }
+        public override string ToString()
+        {
+            string shelfs = "";
+            foreach (Shelf shelf in Shelves)
+            {
+                shelfs += "\n=================================================================================\n";
+                shelfs += "\n" + shelf.ToString();
+            }
+            return "Refrigerator id: " + id + " Model: " + Model + " Color: " + Color + " with " + NumberOfShelfs + " shefls containes \n" + shelfs;
+        }
+        public int SpaceInRefrigerator()
+        {
+            int spaceSum = 0;
+            foreach (Shelf shelf in Shelves)
+                spaceSum += shelf.Space;
+            return spaceSum;
+        }
+        public bool AddItem(Item item)
+        {
+            int count = 0;
+            foreach (Shelf shelf in Shelves)
+            {
+                if (shelf.IsSpaceInShelf(item.Space))
+                {
+                    shelf.AddItem(item);
+                    return true;
+                }
+                else
+                {
+                    if (item.Space > shelf.ShelfSpace)
+                        count++;
+                }
+
+            }
+            if (count == this.Shelves.Count)
+                Console.WriteLine("There is no shelf with this amount of space this item is too big");
```

Comment:
you may replace it to ternary operator ?:

Console.WriteLine(count == Shelves.Count
    ? "There is no shelf with this amount of space this item is too big"
    : "There is no more space in Refrigerator, you might want to clean refrigerator!");